### PR TITLE
docs: Add new vocabulary in naming of functions

### PR DESCRIPTION
**Pull request by cballevre**
In a front team meeting, we chose to include more vocabulary to name function and variable which is mainly use in your apps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,15 +44,29 @@
...
+
+`hasSomething` or `isSomething` : should describe a boolean variable
+
+`computeSomething` : create something from other elements
+
+`makeSomething` : create something from scratch
+
+`doSomethingAndForget` : to convert an async func `doSomething` to a sync one `doSomethingAndForget`
+
+`format` : from an object, returns a string
```
Comment (Crash--) [inline README.md:67]: why not... toString()? Because `format` is too much generic to only return a string

File: README.md [hunk 2]
```diff
@@ -44,15 +44,29 @@
...
 
-`fetchSomething` : should access data by making a server side call
+`findSomething` : like the get, but with arguments
 
 `normalizeSomething` : should transform data to conform to a given schema or to make sure that is coherent with other documents
 
-`harmonizeSomething` : should tweak data a bit to work well with other items
+`saveSomething` : creates or updates a document
+
+`updateSomething` : should perform server-side updates
```
Comment (Crash--) [inline README.md:57]: Since cozy-client doesn't expose any `update` method, I'm not sur about this one.

File: README.md (hunk 1 above)
Comment (JF-Cozy) [reply README.md:67]: on s'en sert par exemple parfois pour formater des dates. A mon avis sauf pour les isToto ou hasToto qui retourne la plupart du temps des bool, on peut éviter dans cette doc de "forcer" un type concernant le retour

Comment (cballevre) [reply README.md:67]: we could simply use the `computeSomething` instead of `formatSomething` if we do not force the return type to string

Comment (cballevre) [reply README.md]: we decide to remove `format` but if you use it often in the future don't hesitate to open a PR

**Pull request merged**